dashboards/google-vertex-ai/vertex-ai-provisioned-throughput.json

```diff
@@ -0,0 +1,314 @@
+{
+  "displayName": "Provisioned Throughput Usage and Limit",
```

Comment:
I think this should be synced with the `display_name` in metadata.yaml for consistency ("Vertex AI Provisioned Throughput").